### ISSUE TITLE
The code for the supervision transfer part

### ISSUE THREAD

**Issue by haochen593** [s-gupta/fast-rcnn#11]
Dear gupta,

Thanks sincerely for your release. 

Since this work allows training modal-specific representations in the target data with limited training samples, I want to use it to my RGB-D system. However, I have tried it with a large variety of training settings, but the Euclidean loss fails to converge all the time.

So, at your convenience, could you help me to find the corresponding training code for the supervision transfer part. I search it carefully but still can not find it. The code will help me a lot.

I will be pretty grateful for your help.

Thanks for your time.
